modules/ROOT/pages/dw-operators.adoc

```diff
@@ -370,3 +367,359 @@ xref:dataweave-variables.adoc#example_local_variable[Examples: Local DataWeave V
+
+The following example updates an attribute value and changes it to upper case:
+
+.Input
+[source,xml,linenums]
+----
+<user name="leandro"/>
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload update {
+    case name at .user.@name -> upper(name)
+}
+----
+
+.Output
+[source,xml,linenums]
+----
+<?xml version='1.0' encoding='UTF-8'?>
+<user name="LEANDRO"/>
+----
+
+==== Dynamic Selector
+
+Any of the given selectors can use an expression to make it dynamic. +
+
+The following example selects and updates a value where the field to be updated is in a variable and not hardcoded. In this case from the variable `theFieldName` with value `name`:
+
+.Input
+[source,json,linenums]
+----
+{
+    "name": "Ken",
+    "lastName":"Shokida"
+}
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+var theFieldName = "name"
+---
+payload update {
+    case s at ."$(theFieldName)" -> "Shoki"
+}
+----
+
+.Output
+[source,json,linenums]
+----
+{
+  "name": "Shoki",
+  "lastName": "Shokida"
+}
+----
+
+=== Conditional Update
+
+The update operator has a syntax to write a conditional update if you only want to modify a field under a given condition. +
+
+The following example updates the category based on the name of the user:
+
+.Input
+[source,json,linenums]
+----
+[{"name": "Ken", "age": 30}, {"name": "Tomo", "age": 70}, {"name": "Kajika", "age": 10}]
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload map ((user) ->
+    user update {
+        case name at .name if(name == "Ken") -> name ++ " (Leandro)"
+        case name at .name if(name == "Tomo") -> name ++ " (Christian)"
+    }
+)
+----
+
+
+.Output
+[source,json,linenums]
+----
+[
+  {
+    "name": "Ken (Leandro)",
+    "age": 30
+  },
+  {
+    "name": "Tomo (Christian)",
+    "age": 70
+  },
+  {
+    "name": "Kajika",
+    "age": 10
+  }
+]
+----
+
+=== Upserting
+
+The update operator allows to insert a field if it is not present using the `!` symbol at the end of selector expression. When the field is not present the value `null` is going to be bound to the variable. +
+
+The following example updates the category based on the name of the user:
+
+.Input
+[source,json,linenums]
+----
+[{"lastName": "Doe"}, {"lastName": "Parker", "name": "Peter" }]
+----
+
+.Source
+[source,weave,linenums]
+----
+%dw 2.0
+output application/json
+---
+payload  map ((user) ->
+    user update {
+        case name at .name! -> if(name == null) "JOHN" else upper(name)
+    }
+)
+----
+
+.Output
+[source,json,linenums]
+----
+[
+  {
+    "lastName": "Doe",
+    "name": "JOHN"
+  },
+  {
+    "lastName": "Parker",
+    "name": "PETER"
+  }
+]
+----
+
+
+=== Sugar Syntax
+
+The update operator allows to write a shorter version syntax to avoid some boilerplate code by using a default variable named `$`. +
+
+The following example uses the same Value selector example but using simpler syntax:
```

Comment:
```suggestion
The following example illustrates how to simplify the syntax used for value selector:
```
I was not sure which example provided earlier was "the same Value selector example." As always, tweak suggested wordings to maintain tech accuracy.